mix.exs

```diff
@@ -40,14 +40,8 @@ defmodule GRPC.Mixfile do
       {:jason, ">= 0.0.0", optional: true},
       {:cowlib, "~> 2.12"},
       {:castore, "~> 0.1 or ~> 1.0", optional: true},
-      {:protobuf, "~> 0.11"},
+      {:protobuf, "~> 0.14"},
       {:protobuf_generate, "~> 0.1.1", only: [:dev, :test]},
-      {:googleapis,
-       github: "googleapis/googleapis",
-       branch: "master",
-       app: false,
-       compile: false,
-       only: [:dev, :test]},
```

Comment:
This is the main point of the update.
Should we also regenerate the example protos in the newer version?

Reply:
Yes, I believe that would be ideal.